core/rawdb/accessors_metadata_ext.go

```diff
@@ -0,0 +1,114 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package rawdb
+
+import (
+	"time"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/ethdb"
+	"github.com/ava-labs/libevm/rlp"
+)
+
+// writeTimeMarker writes a marker of the current time in the db at `key`
+func writeTimeMarker(db ethdb.KeyValueStore, key []byte) error {
+	data, err := rlp.EncodeToBytes(uint64(time.Now().Unix()))
+	if err != nil {
+		return err
+	}
+	return db.Put(key, data)
+}
+
+// readTimeMarker reads the timestamp stored at `key`
+func readTimeMarker(db ethdb.KeyValueStore, key []byte) (time.Time, error) {
+	data, err := db.Get(key)
+	if err != nil {
+		return time.Time{}, err
+	}
+
+	var lastRun uint64
+	if err := rlp.DecodeBytes(data, &lastRun); err != nil {
+		return time.Time{}, err
+	}
+
+	return time.Unix(int64(lastRun), 0), nil
+}
+
+// deleteTimeMarker deletes any value stored at `key`
+func deleteTimeMarker(db ethdb.KeyValueStore, key []byte) error {
```

Comment:
do we need this? It's fine if we want to keep CRUD but seems it just calls delete on the db.

Reply:
Definitely does not look like we need this, removed.